.github/workflows/run-cluster.yaml

```diff
@@ -45,12 +41,20 @@ on:
         default: ""
 
 jobs:
+  build-commit:
+    uses: ./.github/workflows/build-commit.yaml
+    with:
+      arch: x86
+      python_version: ${{ inputs.python_version }}
+    secrets:
+      ACTIONS_AWS_ROLE_ARN: ${{ secrets.ACTIONS_AWS_ROLE_ARN }}
+
   run-command:
     runs-on: [self-hosted, linux, x64, ci-dev]
-    timeout-minutes: 15 # Remove for ssh debugging
```

Comment:
Seems you forgot to put this back in

Reply:
Actually, I removed it on purpose. Certain scale-factors and questions take over 15min. Here is a workflow that was proceeding successfully until the timeout was hit and everything else was cancelled:

https://github.com/Eventual-Inc/Daft/actions/runs/12401178483